purchase_blanket_order/tests/test_purchase_order.py

```diff
@@ -157,7 +157,7 @@ def test_01_create_purchase_order(self):
             }
         )
         po_line = po.order_line[0]
-        po_line.with_context(from_purchase_order=True).name_get()
+        po_line.with_context(from_purchase_order=True)._compute_display_name()
```

Comment:
curious: should it not be auto-triggered -- as computed -- based on depends...?

though it should not hurt anything, calling explicilty :)

Reply:
my guess is that `name_get` was different mechanism -- not involving `computed` -- so needed manual call...